main/.vuepress/config.js

```diff
@@ -100,9 +100,9 @@ module.exports = {
           collapsable: false,
           sideBarDepth: 3,
           children: [
-            '/zoe/guide/contracts/autoswap',
+            '/zoe/guide/contracts/auto-swap',
```

Comment:
We've been using 'autoswap' as one word without the hyphen. Let's stick with the old name for this file for conformity.